app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,49 @@
+$font-color: #088da5;
+$black: #000000;
+
+body {
+  background-color: $black;
+}
+
+h1,
+h3 {
+  color: $font-color;
+  text-align: center;
+}
+
+.about {
+  border: 3px solid $black;
+  text-align: center;
```

Comment:
eh you can ignore Hound here - I almost never say that, but Hound's CSS comments are annoying as *&!#